manifest.json

```diff
@@ -6,6 +6,15 @@
   "permissions": [
     "tabs"
   ],
+  "commands":{
+    "close": {
+      "description":"Close duplicate tabs"
+    },
+    "_execute_action":{
+      "suggested_key": "Alt+U"
```

Comment:
Is it cross-platform compatible?

Reply:
![Screenshot 2024-09-15 at 9 23 36 PM](https://github.com/user-attachments/assets/9c0fea88-4f39-4117-a8c5-a60482b90900)
From a VM running Catalina. Can confirm that "Option"+U  opens the close tabs button.

Reply:
Thanks, @Michael-Gallo! It works now! It seems it didn't load correctly the first time. I've submitted the new version for review on the Chrome Developer Dashboard.